apps/mobile/src/components/TradeChecklistFlow/components/AgreeOnTradeDetailsScreen/components/OnlineOrInPersonTrade.tsx

```diff
@@ -78,8 +101,15 @@ function OnlineOrInPersonTrade(): JSX.Element {
             {tradeChecklistItems.map((item) => (
               <ChecklistCell
                 key={item}
-                itemStatus={'unknown'}
+                itemStatus={
+                  mainTradeCheckListScreenState.find(
```

Comment:
It would be easier if we could just do this: 

```suggestion
                  mainTradeCheckListScreenState[state.type]
```
it seems to me like we are not gaining anything by using an array